packages/epics/src/people/hooks/validation.ts

```diff
@@ -11,3 +11,21 @@ export const purchaseSchema = z.object({
     .min(1, { message: 'Please add a recipient or wallet address' })
     .refine(isAddress, { message: 'Invalid Ethereum address' }),
 });
+
+export const activateSpacesSchema = z.object({
+  spaces: z
+    .array(
+      z.object({
+        spaceId: z.number().min(1, 'Space is required'),
+        months: z.number().min(1, 'Months must be at least 1'),
```

Comment:
_optional_
This check is fine but can be enhanced with the integer constraint
```suggestion
        spaceId: z.number().int().min(1, 'Space is required'),
        months: z.number().int().min(1, 'Months must be at least 1'),
```